src/services/browser/server.ts

```diff
@@ -1,100 +1,29 @@
 import { BrowserWindow } from 'electron'
```

Comment:
This is a good example of how much simpler the implementation is with the new messaging system.